rules/falco_rules.yaml

```diff
@@ -3398,6 +3398,37 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+#The given rule is designed to detect suspicious activities related to disabling system recovery features. 
+#The rule looks for specific commands being executed such as rm -rf, chattr +i, mkfs.ext4 -E nodiscard, mount remount,ro, 
+#and disabling certain services such as systemd-backlight@.service, apport.service, rescue.service, emergency.service, and recovery-mode.service. 
+#https://attack.mitre.org/techniques/T1490/
+#Mitre Impact: Inhibit system recovery
+
+- list: recovery_files
+  items: [/etc/default/grub, /etc/inittab, /usr/lib/systemd/system/rescue.service]
+  
+- list: critical_dir
+  items: [/boot, /bin, /etc, /lib, /sbin, /usr, ~/.local/share/Trash, ~/.local/share/Trash/files]
+
+- list: recovery_proc_args
+  items: ["disable systemd-backlight@.service", "disable apport.service", "disable rescue.service", "disable emergency.service", "disable recovery-mode.service"]
+
+- rule: Disable Recovery Features
+  desc: Detects disabling system recovery features by deleting or disabling services and commands.
+  condition: >
+      (spawned_process) and 
+      ((proc.name = "chattr" and proc.args contains "+i") or
+      (proc.name = "mkfs.ext4" and proc.args contains "-E nodiscard") or
+      (proc.name = "mount" and proc.args contains "remount,ro") or
+      (proc.name = "systemctl" and proc.args in (recovery_proc_args)) or
+      (open_write and fd.name in (recovery_files)) or
+      (remove and (fd.name pmatch (critical_dir))))
+  enabled: true
```

Comment:
Can you also remove the enabled:true?
If it's true we don't need to specify this.